src/test/scala/calc/RunTest.scala

```diff
@@ -18,7 +19,7 @@ import org.scalajs.jsenv.JSConsole
  *  result.
  */
 class RunTest {
-
+  val r = Random
```

Comment:
Sure. I got it! Since it's just a prototype, I did not care some corner cases. Just.. hmm, works on some common cases